### ISSUE TITLE
Silent error on wrong path

### ISSUE THREAD

**Issue by crystalthoughts** [cumulo-autumn/StreamDiffusion#45]
I'm struggling to set an absolute path in the demo to an already downloaded model. I get a 'Triton not found' error but then nothing, when I cancel the process it shows an exception noting that I've supplied an invalid path (I'm on windows).

What is the workflow for pointing to a downloaded model? The included models directory doesn't seem to be used anywhere...

**Comment by liqMix** [comment]
The Triton errors seem to be fine if you're on Windows, they haven't affected my ability to use the tool.

I ran into similar issues. I ended up modifying a few places and got some local models working. Don't know if there's something I missed though that makes these changes unnecessary.
Also, this is for `examples/screen` but may be similar for other demos.

I set this value
https://github.com/cumulo-autumn/StreamDiffusion/blob/62596fd39b86a3088e28b2f2c58caa36bc069c50/examples/screen/main.py#L197
To my local model path:
```
    model_id_or_path: str = "F:\AI\img\StreamDiffusion\models\Model\\turbo.safetensors", 
```

And then adjusted the wrapper a little bit.
https://github.com/cumulo-autumn/StreamDiffusion/blob/62596fd39b86a3088e28b2f2c58caa36bc069c50/utils/wrapper.py#L410-L413

Modified that to try `from_single_file` first:
```
        try:  # Load from local directory
            pipe: StableDiffusionPipeline = StableDiffusionPipeline.from_single_file(
                model_id_or_path,
            ).to(device=self.device, dtype=self.dtype)
```

**Comment by Ryslik777** [comment]
Thank you for your contribution.

**Comment by crystalthoughts** [comment]
Yes thanks - i didn’t have much luck with your advice so I’ll have to dig
into it a bit more
